src/cmake/axom-config.cmake.in

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#------------------------------------------------------------------------------
+
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+
+@PACKAGE_INIT@
+
+if(NOT AXOM_FOUND)
+
+  #----------------------------------------------------------------------------
+  # Set version and paths
+  #----------------------------------------------------------------------------
+  
+  set(AXOM_VERSION "@AXOM_VERSION_FULL@")
+  
+  set(AXOM_INSTALL_PREFIX "@AXOM_INSTALL_PREFIX@")
+  set(AXOM_INCLUDE_DIRS "${AXOM_INSTALL_PREFIX}/include/axom")
```

Comment:
Remove `axom` at the end, since in applications we prefix all of axom includes with `axom`. This would also make available the bundled TPLs that we install.

Reply:
Thanks @gzagaris -- I agree, the `axom` part doesn't belong there.